c/misra/test/rules/RULE-21-13/CtypeFunctionArgNotUnsignedCharOrEof.expected

```diff
@@ -0,0 +1,2 @@
+| test.c:17:7:17:17 | call to islower | The <ctype.h> function call to islower accepts an argument c3 that is not an unsigned char nor an EOF. |
+| test.c:23:7:23:17 | call to isdigit | The <ctype.h> function call to isdigit accepts an argument c4 that is not an unsigned char nor an EOF. |
```

Comment:
This expected results file needs updating after you changed the test case.

Reply:
👍